src/components/aeRoundedContainer/aeRoundedContainer.js

```diff
@@ -0,0 +1,54 @@
+import {TYPE_PROPERTY_VALUES as types} from '../../constants'
+
+export default {
+  name: 'ae-rounded-container',
+  props: {
+    type: {
+      type: String,
+      default: 'normal',
+      validator: value => types.includes(value)
+    },
+    direction: {
+      type: String,
+      default: 'row',
+      validator: value => ['row', 'row-reverse', 'column', 'column-reverse'].includes(value)
+    },
+    wrap: {
+      type: String,
+      default: 'nowrap',
+      validator: value => ['nowrap', 'wrap', 'wrap-reverse'].includes(value)
+    },
+    justifyContent: {
+      type: String,
+      default: 'start',
+      validator: value => [
+        'start', 'end', 'center', 'space-between', 'space-around', 'space-evenly'
+      ].includes(value)
+    },
+    alignItems: {
+      type: String,
+      default: 'stretch',
+      validator: value => ['start', 'end', 'center', 'baseline', 'stretch'].includes(value)
+    },
+    alignContent: {
+      type: String,
+      default: 'stretch',
+      validator: value => [
+        'start', 'end', 'center', 'space-between', 'space-around', 'stretch'
+      ].includes(value)
+    }
```

Comment:
Even in `scoped` mode, it is possible to set styles of the root node of child components, Is it necessary to wrap possible css properties with vue component properties with so lot of code?